tests/helpers/test_config_validation.py

```diff
@@ -1060,8 +1060,30 @@ class TestEnum(enum.Enum):
 
     schema = vol.Schema(cv.enum(TestEnum))
 
-    with pytest.raises(vol.Invalid):
-        schema("value3")
+    for value in ("Value 1", "value3", TestEnum.value1, None):
+        with pytest.raises(vol.Invalid):
+            schema("value3")
+
+    schema("value1")
+
+
+def test_enum_value():
+    """Test enum_value validator."""
+
+    class TestEnum(enum.Enum):
+        """Test enum."""
+
+        value1 = "Value 1"
+        value2 = "Value 2"
+
+    schema = vol.Schema(cv.enum_value(TestEnum))
+
+    for value in ("value1", "Value 3", None):
+        with pytest.raises(vol.Invalid):
```

Comment:
Validate that the error message is correct. 

https://github.com/home-assistant/core/blob/dev/tests/util/test_json.py#L76-L82